docs/copilot/copilot-extensibility-overview.md

```diff
@@ -10,34 +10,46 @@ MetaSocialImage: images/shared/github-copilot-social.png
+To extend Copilot in VS Code, you build a VS Code extension with additional capabilities, including:
 
-You have different options to extend GitHub Copilot in your VS Code extension:
+- Adding a chat participant in the VS Code Chat view to give users a natural-language interface
+- Using the Copilot LLM to interpret user prompts and generate responses
+- Using the Copilot LLM to implement AI-powered features in your extension
 
-- Use the [Chat API](/api/extension-guides/chat.md) to create a **chat participant** to give users a natural-language interface and provide domain-specific help via the Chat view in VS Code.
+## About Copilot extensibility in VS Code
 
-- Use the [**Language Model API**](/api/extension-guides/language-model.md) and the VS Code extension API to build AI-powered features into VS Code.
+You have different options to extend Copilot in your VS Code extension:
 
-With GitHub Copilot, you can expand the capabilities of your extension by using the power of AI and give developers a richer and more productive experience.
+- **VS Code Chat extension**: use the Chat API to create a chat participant that gives users a natural-language interface and provides domain-specific help via the Chat view in VS Code. Optionally, you can use the Copilot LLM to interpret user prompts and generate responses.
 
-## Get started
+- **Use Copilot's LLM**: use the Language Model API and the VS Code extension APIs to build AI-powered features into your extension and enhance editor-specific interactions.
 
-Get started with GitHub Copilot extensibility by taking the following tutorials:
+Alternatively, you can also build a **Copilot Extension**, implemented as a GitHub App with additional capabilities. Copilot Extensions work across all supported IDEs and GitHub, but don't have access to functionalities specific to VS Code. Get more info about [Copilot Extensions](https://docs.github.com/en/copilot/building-copilot-extensions/about-building-copilot-extensions) in the GitHub documentation.
 
-- [Tutorial: Create a code tutor chat participant with the Chat API](/api/extension-guides/chat-tutorial.md)
-- [Tutorial: Generate code annotations by using the Language Model API](/api/extension-guides/language-model-tutorial.md)
+## Use cases
 
-## Related content
+You can extend the capabilities of Copilot in VS Code in various ways. For example, you could use them for:
+
+- Docs querying: A chat participant could allow Copilot Chat to query a third-party documentation service to find information about a specific topic.
+- AI-assisted coding: Use the Copilot LLM to provide editor annotations to provide coding suggestions.
+- AI-powered reviews: Use the Copilot LLM to review your code for security vulnerabilities or performance improvements.
+- Data retrieval: A chat participant could allow Copilot Chat to query a database or third-party data service to retrieve information about a specific topic.
+
```

Comment:
Maybe add two more bullet points: "Enterprise coding assistant: a chat participant that is grounded in the data of your enterprise, and that is aware of specific coding guidelines your company follows."
"Expert in a specific area, for example payment processing"

Reply:
And last bullet point: "you existing VS Code extension can now easily be AI-powered by using the language model API"